pulp_rpm/app/fields.py

```diff
@@ -31,7 +33,7 @@ def to_representation(self, obj):
                 'release': pkg['release'],
                 'src': pkg['src'],
                 'sum': pkg['sum'],
-                'sum_type': pkg['sum_type'],
+                'sum_type': get_key_by_value(ADVISORY_SUM_TYPES, pkg['sum_type']),
```

Comment:
This is not going to be an efficient operation, there should be a separate lookup table for going int -> str.

Reply:
Do you mean to have two dicts? Each for one lookup operation?

Reply:
Yes